composer.json

```diff
@@ -13,9 +13,9 @@
     ],
     "homepage": "http://github.com/bshaffer/oauth2-server-httpfoundation-bridge",
     "require":{
-        "php":">=5.3.0",
+        "php":">=7.0.0",
```

Comment:
This looks good, but we'll want to change the `.travis.yaml` to reflect this by removing testing on PHP 5.x

Reply:
Hmm, I did another check with PHPUnit, and I think we need to bump it even up to 7.2. PHP versions lower than 7.2 throw the following error:

`OAuth2\HttpFoundationBridge\ResponseTest::testSetRedirect with data set #4 ('http://test.com/path?state=xy...nt.com', 'http://test.com/path', 'xyz', 'foo', 'this is a description', 'http://brentertainment.com')
TypeError: Return value of OAuth2\HttpFoundationBridge\Response::setStatusCode() must be an instance of OAuth2\HttpFoundationBridge\object, instance of OAuth2\HttpFoundationBridge\Response returned`

Reply:
You added `public function setStatusCode($statusCode, $text = null): object`, which is causing this error. That's not correct.

Reply:
It seems you're right, we cant bump this to Http Foundation 5.0 unless we bump the whole library to PHP 7.2